src/patternfly/components/Page/page.scss

```diff
@@ -145,26 +143,29 @@
 .pf-c-page__header {
   @include pf-t-dark; // force the container to follow the dark theme
 
-  grid-area: header;
   z-index: var(--pf-c-page__header--ZIndex);
-  display: flex;
-  flex-wrap: wrap;
+  grid-template-columns: auto auto;
+  display: grid;
+  grid-area: header;
   align-items: center;
-  max-width: 100%;
   min-height: var(--pf-c-page__header--MinHeight);
 
   > * {
     display: flex;
     align-items: center;
   }
+
+  @media screen and (min-width: $pf-global--breakpoint--lg) {
+    grid-template-columns: auto 1fr auto;
+  }
 }
 
 // Brand
 .pf-c-page__header-brand {
+  grid-column: 1 / 2;
   padding-left: var(--pf-c-page__header-brand--PaddingLeft);
 
   @media (min-width: $pf-global--breakpoint--md) {
-    flex: 0 0 var(--pf-c-page__header-brand--md--FlexBasis); // Hardcode header brand width based on design spec, must match --pf-c-nav--Width
     align-self: stretch;
```

Comment:
do you still need this without flex?

Reply:
It's a grid property, too, though I don't think it's necessary. I think it was there since this was the element that had the semi-opaque dark background color.